src/app/model/user/profil.ts

```diff
@@ -0,0 +1,15 @@
+export class Profil {
+
+    constructor(
+        public id: number,
+        public firstname: string,
+        public lastname: string,
+        public email: string,
+        public password: string,
+        public role: string,
```

Comment:
An User can be in multiple different teams, it's not a good idea to put role in Profil class since those values will be differrent for each team